symbols.css

```diff
@@ -132,11 +156,40 @@ a {
   width: 100%;
   overflow: hidden;
 }
+.symbol > .name,
+.symbol.clicked > .copy {
+  display: block;
+}
+
+.symbol.clicked > .name,
+.symbol > .copy {
+  display: none;
+}
+
+
+.symbol > .glyph > input {
+  color: var(--main-fg-color);
+  background-color: var(--main-bg-color);
+}
 
 .symbol:hover {
   background-color: var(--hover-color);
 }
 
+.symbol:hover > .remove:after {
+  color: var(--warning-fg-color);
+  content: "🗑";
```

Comment:
In situ it's not immediately obvious to me that this is a trash can, could we try ❌ instead?

Reply:
Whatever you want. Also in 914415c9dd6387c7983bcf6ddd23508b77b5c2fe